modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,94 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, the auto-instrumentation configures only the OTLP Exporter. Due to this, you must configure the endpoint to point to the OTLP Receiver on the Collector. You can do so by using a config map as well as a Secret for the TLS and CA certificates.
+
+.Sample exporter configuration using a config map
```

Comment:
The config map is only there to configure TLS CA.

I would rename it to ".Sample exporter TLS CA configuration using a config map"

Reply:
> The config map is only there to configure TLS CA.

@pavolloffay, do you want me to keep this line also/alternatively in the Secret? I'm asking because your draft mentioned both: https://github.com/openshift/openshift-docs/pull/83387/files#diff-e469e721412bda971611ec4d0210f323895fded0c21ab21415c685210e9d3ab2R121.